2dcontext/imagebitmap/common.js

```diff
@@ -38,6 +38,7 @@ function makeVideo() {
         video.oncanplaythrough = function() {
             resolve(video);
         };
+        video.onerror = reject;
```

Comment:
Looks good to add onerror in any case.
Test might fail in WebKit as the test tries to load ogv.
Can we use common/media.js to abstract this test of specific video formats?

Reply:
Look for getVideoURI for examples

Reply:
I think the needed pattern.mp4 doesn't exist yet; someone would need to create it first.